lib/matplotlib/transforms.py

```diff
@@ -1981,9 +1980,14 @@ def scale(self, sx, sy=None):
         """
         if sy is None:
             sy = sx
-        scale_mtx = np.array(
-            [[sx, 0.0, 0.0], [0.0, sy, 0.0], [0.0, 0.0, 1.0]], float)
-        self._mtx = np.dot(scale_mtx, self._mtx)
+        # Writing out the slice drastically improves
```

Comment:
This comment explains the change from the previous code, but it's not self-contained for a future reader (what slice?). Maybe something like
```suggestion
        # explicit element-wise scaling is fastest
```